cmd/ci-operator-prowgen/main.go

```diff
@@ -195,8 +195,8 @@ func writeToDir(dir string, c map[string]*prowconfig.JobConfig) error {
 	errCh := make(chan error)
 	map_ := func() error {
 		for x := range ch {
-			i := strings.Index(x.k, "/")
-			org, repo := x.k[:i], x.k[i+1:]
+			i := strings.Split(x.k, "/")
+			org, repo := i[0], i[1]
```

Comment:
The only thing that was changed here is that the code now will panic if there is no `/`